source/references.rst

```diff
@@ -26,3 +26,6 @@ References
 .. [TFAFFAMB] `TF-A Secure Partition Manager: FF-A manifest binding to device tree <https://trustedfirmware-a.readthedocs.io/en/latest/components/ffa-manifest-binding.html>`
 
 .. [TCG_EFI] `https://trustedcomputinggroup.org/resource/tcg-efi-protocol-specification`.
+
+.. [DICE] `https://trustedcomputinggroup.org/wp-content/uploads/DICE-Layering-Architecture-r19_pub.pdf`.
+.. [Open-DICE] `https://pigweed.googlesource.com/open-dice/+/refs/heads/main/docs/specification.md`.
```

Comment:
Please fix the hyperlink syntax in these references. See https://github.com/FirmwareHandoff/firmware_handoff/pull/39 that fixes the other links.

Reply:
done in [fd0e476](https://github.com/FirmwareHandoff/firmware_handoff/pull/38/commits/fd0e476b924f77c34e6694569b2f369a4cd97542)